### PR TITLE
Panorama playbooks - DAG, EDL, new commit configuration playbook

### PR DESCRIPTION
**Pull request by Arsenikr**
PAN-OS DAG
PAN-OS EDL
PAN-OS commit configuration playbook - deprecating the old `PanoramaCommitConfiguration`.

## Status
Ready

## Related Issues
fixes: 

## Description
- playbook for setting up and adding an ip to a DAG in PAN-OS
- playbook for EDL setup in PAN-OS
- new commit configuration playbook with automatic product type detection

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation - after CR
- [x] Code Review

### CODE REVIEW

File: Playbooks/playbook-PAN-OS_DAG_Configuration.yml [hunk 1]
```diff
@@ -0,0 +1,434 @@
+id: a893e34e-f38a-4dc2-8e7e-fd900d18f531
```
Comment (bakatzir) [inline Playbooks/playbook-PAN-OS_DAG_Configuration.yml]: id should be to equal to the name

File: Playbooks/playbook-PanoramaCommitConfiguration.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: -1
 name: PanoramaCommitConfiguration
 fromversion: 4.0.0
 description: "Commit PAN-OS Panorama or Firewall Configuration.\nIf specified
-  as Panorama, will also push the Policies to the specified Device Group in the instance "
+  as Panorama, will also push the Policies to the specified Device Group in the instance. playbook is deprecated please use pan-os-commit-configuration instead"
```
Comment (bakatzir) [inline Playbooks/playbook-PanoramaCommitConfiguration.yml]: please add the deprecated tag

File: TestPlaybooks/playbook-PAN-OS_DAG_Configuration_Test.yml [hunk 3]
```diff
@@ -0,0 +1,634 @@
+id: 3114a5ac-942f-4766-8ebc-fbec4a188d67
```
Comment (bakatzir) [inline TestPlaybooks/playbook-PAN-OS_DAG_Configuration_Test.yml]: id should be to equal to the name

File: TestPlaybooks/playbook-PAN-OS_DAG_Configuration_Test.yml [hunk 4]
```diff
@@ -0,0 +1,634 @@
...
+version: 23
```
Comment (bakatzir) [inline TestPlaybooks/playbook-PAN-OS_DAG_Configuration_Test.yml]: version should be -1

File: Tests/conf.json [hunk 5]
```diff
@@ -1482,6 +1482,11 @@
             "playbookID": "PAN OS EDL Management - Test",
             "integrations": "palo_alto_networks_pan_os_edl_management"
         },
+        {
+            "playbookID": "playbook-PAN-OS_DAG_Configuration_Test",
+            "integrations": "Panorama",
+            "timeout": 2400
```
Comment (bakatzir) [inline Tests/conf.json]: really needed that high?

**Comment by bakatzir** [comment]
@Arsenikr please also open a relevant git issue and connect with the fixes

File: Playbooks/playbook-PanoramaCommitConfiguration.yml (hunk 2 above)
Comment (Arsenikr) [reply Playbooks/playbook-PanoramaCommitConfiguration.yml]: it failed to commit with the deprecated tag

**Comment by bakatzir** [comment]
@kirbles19 arseny/I would love to go over the playbooks with you regarding not only the descriptions but also the task names, since it's sort of a 1st of it's kind setup playbooks(3).
PB:
https://github.com/demisto/content/blob/09ddcedc54f9c43bdaf64aeb185fa1816eb53f23/Playbooks/playbook-PAN-OS_DAG_Configuration.yml
https://github.com/demisto/content/blob/09ddcedc54f9c43bdaf64aeb185fa1816eb53f23/Playbooks/playbook-PAN-OS_EDL_Setup.yml
https://github.com/demisto/content/blob/09ddcedc54f9c43bdaf64aeb185fa1816eb53f23/Playbooks/playbook-Pan-OS_Commit_Configuration.yml
https://github.com/demisto/content/blob/09ddcedc54f9c43bdaf64aeb185fa1816eb53f23/Playbooks/playbook-PanoramaCommitConfiguration.yml

RNs:
https://github.com/demisto/content/blob/09ddcedc54f9c43bdaf64aeb185fa1816eb53f23/Playbooks/playbook-PAN-OS_DAG_Configuration_CHANGELOG.md
https://github.com/demisto/content/blob/09ddcedc54f9c43bdaf64aeb185fa1816eb53f23/Playbooks/playbook-PAN-OS_EDL_Setup_CHANGELOG.md
https://github.com/demisto/content/blob/09ddcedc54f9c43bdaf64aeb185fa1816eb53f23/Playbooks/playbook-Pan-OS_Commit_Configuration_CHANGELOG.md
https://github.com/demisto/content/blob/09ddcedc54f9c43bdaf64aeb185fa1816eb53f23/Playbooks/playbook-PanoramaCommitConfiguration_CHANGELOG.md

**Comment by bakatzir** [comment]
@kirbles19 Let us know when we can set some time and tend to these playbooks.
we want them for the upcoming release.

**Comment by kirbles19** [comment]
@Arsenikr Finished reviewing .md files. Please check all edits for technical accuracy.

**Comment by kirbles19** [comment]
@Arsenikr @bakatzir all .yml files reviewed. All done.

**Pull request merged**